settings.gradle.kts

```diff
@@ -36,3 +36,4 @@ include("span-normalizer:span-normalizer-constants")
 
 // e2e pipeline
 include("hypertrace-ingester")
+include("telemetry-attribute-utils")
```

Comment:
`telemetry-attribute-utils` -> `attribute-utils` ?

Reply:
yes renamed the module